applicationset/utils/webhook.go

```diff
@@ -202,15 +209,31 @@ func getPRGeneratorInfo(payload interface{}) *prGeneratorInfo {
 			Owner:     payload.Repository.Owner.Login,
 			APIRegexp: apiRegexp,
 		}
+	case gitlab.MergeRequestEventPayload:
+		if !isAllowedGitlabPullRequestAction(payload.ObjectAttributes.Action) {
+			return nil
+		}
+
+		apiURL := payload.Project.WebURL
+		urlObj, err := url.Parse(apiURL)
+		if err != nil {
+			log.Errorf("Failed to parse repoURL '%s'", apiURL)
+			return nil
+		}
+
+		info.Gitlab = &prGeneratorGitlabInfo{
+			Project:     strconv.FormatInt(payload.ObjectAttributes.TargetProjectID, 10),
```

Comment:
Does it good solution for int cast? sorry, I'm not a go developer 😀

Reply:
I think this is fine!